orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/handler/AuthenticationAware.kt

```diff
@@ -56,4 +57,24 @@ interface AuthenticationAware {
       ExecutionContext.clear()
     }
   }
+
+  fun backtrackSkippedStages(stage: StageExecution): StageExecution {
+    if (stage.isManualJudgmentType &&
+      !stage.status.isSkipped &&
+      stage.withPropagateAuthentication()) {
+      return stage;
+    }
+    else {
```

Comment:
you can remove `else` to avoid nested statements

Reply:
Done!, Thank you